plugins/database/src/db_plugin.cpp

```diff
@@ -15810,6 +15810,67 @@ auto db_check_auth_credentials_op(irods::plugin_context& _ctx,
     }
 } // db_check_auth_credentials_op
 
+auto db_execute_genquery2_sql(irods::plugin_context& _ctx,
+                              const char* _sql,
+                              const std::vector<std::string>* _values,
+                              char** _output) -> irods::error
```

Comment:
This function can execute any `SELECT` statement even though the name includes GenQuery2.

Reply:
Not a problem.